packages/fxa-content-server/tests/functional/sign_in_totp.js

```diff
@@ -265,11 +261,9 @@ registerSuite('TOTP - unverified session', {
   beforeEach: function() {
     email = TestHelpers.createEmail('sync{id}');
 
-    return this.remote.then(createUser(email, PASSWORD, { preVerified: true }));
-  },
-
-  afterEach: function() {
-    return this.remote.then(clearBrowserState({ force: true }));
```

Comment:
I can't remember why I have `clearBrowserState` in the `afterEach` and `beforeEach`, maybe being extra cautious 🤷🏽‍♂️